### PR TITLE
Added virtual environment step

### PR DESCRIPTION
**Pull request by ajmaln**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -52,10 +52,14 @@ Copy sample environment file and configure it as per your local settings
...
 ```
 
-Install dependencies
+Setup virtual environment and install dependencies
 
 ```
-pip3 install -r requirements.txt
+pip3 install virtualenv
+virtualenv venv
+source venv/bin/actiavte
```
Comment (kira0204) [inline README.md]: Typo :) 
Also please add the `deactivate` command as well

**Comment by kira0204** [comment]
Also, please make the necessary changes in `.gitignore` as well
```
# Virtualenv
venv
``` should suffice

File: README.md (hunk 1 above)
Comment (joicemjoseph) [reply README.md]: #26 does the same

Comment (tessie) [reply README.md]: typo

**Comment by ajmaln** [comment]
@tessie, the typo has been fixed in my last commit.

**Comment by yeaske** [comment]
@ajmaln Need to solve the merge conflicts.

**Comment by ajmaln** [comment]
Fixed Merge Conflicts

**Comment by jilvin** [comment]
@ajmaln @tessie @abishekvashok Can you check #392?

**Comment by remedcu** [comment]
You may close this PR.

Related ones: #23 #26 #33 #392 

I think just #392 (if required) is enough.

@syam3526 @vigneshhari @biswaz @vrthra

**Pull request closed without merging**